src/components/PeopleManagement/EditGroupNameModal.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ActionRow, Button, Form, ModalDialog,
+} from '@openedx/paragon';
+import { logError } from '@edx/frontend-platform/logging';
+
+import LmsApiService from '../../data/services/LmsApiService';
+
+const EditGroupNameModal = ({
+  group, isOpen, close, setShowToast, setToastMessage,
+}) => {
+  const [name, setName] = useState(group?.name);
+  const editEnterpriseGroup = async () => {
+    try {
+      const formData = { name };
+      const response = await LmsApiService.updateEnterpriseGroup(group?.uuid, formData);
```

Comment:
I believe the url needs to be updated since the patch request is not going through. I think you just need to add a forward slash at the end of the url. we might also want to consider how we want to handle the patch request error. what kind of error do we want to display?

https://github.com/user-attachments/assets/79af3692-6be6-4884-a99d-8e22a1dc0428